ksp_plugin_adapter/main_window.cs

```diff
@@ -239,8 +239,32 @@ internal class MainWindow : SupervisedWindowRenderer {
     }
     UnityEngine.GUI.DragWindow();
   }
+  
+  static bool correct_orientation = true;
+  static bool correct_angular_velocity = true;
+  static bool thresholding = true;
```

Comment:
Could we move these declaration at the place where we normally put data members, i.e., at the end of the class?  Also, explicit `private`.

Another point: it seems to me that these should be persisted in `Load` and `Save`.  As written, when you restart the game you will go back to the default values.  If we believe that users might want to toggle these parameters when they run into trouble, making them non-persistent is going to be horribly confusing.

Reply:
We really don’t want users to be playing with this; many configurations lead to absurd behaviour.
We want to keep playing with it though (and we don’t want to persist them to their non-default state, we that hid a bug during Frobenius development already…).